print-level-order/__test__/print_level_order.test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
Good tests, but needs one more test assertion to get full points on the test assertion portion.